transactions/src/main/resources/subsystem-templates/transactions.xml

```diff
@@ -9,6 +9,7 @@
            </process-id>
        </core-environment>
        <recovery-environment socket-binding="txn-recovery-environment" status-socket-binding="txn-status-manager"/>
+       <object-store relative-to="jboss.server.data.dir"/>
```

Comment:
Is this required? For example is it possible to use a standalone.xml that does not have this attribute as some object store types (e.g. jdbc object store) do not use "relative-to".